core/templates/dev/head/base.html

```diff
@@ -225,9 +225,35 @@
                             </ul>
                           </li>
                         {% endif %}
-                      </ul>
-                    </div>
-                  {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if="windowWidth > 700"></create-exploration-button>
+                          <li ng-if="windowWidth > 700" class="dropdown oppia-clickable-dropdown pull-right" >
+                            <a class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;" href="/forum">
```

Comment:
The oppia-navbar-profile-dropdown-toggle class name doesn't seem appropriate for how it's being used here. It might be better to create a new class altogether (that also includes the custom styles you've added to this element).